From 9b6e53636133fc0ddcabf49dec8a964ef60b59f9 Mon Sep 17 00:00:00 2001
From: Po Liu <Po.Liu@nxp.com>
Date: Wed, 11 Dec 2019 16:58:00 +0800
Subject: [PATCH 11/15] demo:cnc: limit Qci config index with DEC values

Yang model only support DEC value. Limit to DEC input.

Signed-off-by: Po Liu <Po.Liu@nxp.com>
---
 demos/cnc/templates/configQci.html | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/demos/cnc/templates/configQci.html b/demos/cnc/templates/configQci.html
index 4a77ba0..c05f5b9 100644
--- a/demos/cnc/templates/configQci.html
+++ b/demos/cnc/templates/configQci.html
@@ -278,7 +278,7 @@ var sidconf = '<div id="sidconf"><br>&nbsp;&nbsp;' +
 	'<input class="button addlist" type="radio" name="enable" value="true" onclick="selectenable()"  checked="checked">enable'+
 	'&nbsp;<input class="button addlist" type="radio" name="enable" value="false" onclick="selectenable()">disable' +
 	'<br>&nbsp;&nbsp;*index:'+
-	'<input class="button value" type="text" name="index" id="index">'+
+	'<input class="button value" type="text" name="index" id="index" onkeydown=octkeydown(this) onkeyup=octkeyup(this)>'+
 	'<br>&nbsp;&nbsp;*stream handle:'+
 	'<input class="button value" type="text" name="streamhandle" id="streamhandle" onkeydown=octkeydown(this) onkeyup=octkeyup(this)>'+
 	'<br>&nbsp;&nbsp;*filter type:'+
@@ -306,7 +306,7 @@ var sficonf = '<div id="sficonf"><br>&nbsp;&nbsp;' +
 	'<input class="button addlist" type="radio" name="enable" value="true" onclick="selectenable()"  checked="checked">enable'+
 	'&nbsp;<input class="button addlist" type="radio" name="enable" value="false" onclick="selectenable()">disable' +
 	'<br>&nbsp;&nbsp;*index:'+
-	'<input class="button value" type="text" name="index" id="index">'+
+	'<input class="button value" type="text" name="index" id="index" onkeydown=octkeydown(this) onkeyup=octkeyup(this)>'+
 	'<br>&nbsp;&nbsp;*stream handle:'+
 	'<input class="button value" type="text" name="streamhandlesfi" id="streamhandlesfi" onkeydown=octkeydown(this) onkeyup=octkeyup(this)>'+
 	'<br>&nbsp;&nbsp;*stream gate index:'+
@@ -321,7 +321,7 @@ var sgiconf = '<div id="sgiconf"><br>&nbsp;&nbsp;'+
 	'<input class="button addlist" type="radio" name="enable" value="true" onclick="selectenable()"  checked="checked">enable'+
 	'&nbsp;<input class="button addlist" type="radio" name="enable" value="false" onclick="selectenable()">disable' +
 	'<br>&nbsp;&nbsp;*index:'+
-	'<input class="button value" type="text" name="index" id="index">'+
+	'<input class="button value" type="text" name="index" id="index"  onkeydown=octkeydown(this) onkeyup=octkeyup(this)>'+
 	'<br>&nbsp;&nbsp;&nbsp;initial gate state:'+
 	'<input class="button addlist" type="radio" name="initgate" value="open" onclick="selectinitgate()"  checked="checked">open'+
 	'&nbsp;<input class="button addlist" type="radio" name="initgate" value="closed" onclick="selectinitgate()">close' +
@@ -342,7 +342,7 @@ var fmiconf = '<div id="fmiconf"><br>&nbsp;&nbsp;'+
 	'<input class="button addlist" type="radio" name="enable" value="true" onclick="selectenable()"  checked="checked">enable'+
 	'&nbsp;<input class="button addlist" type="radio" name="enable" value="false" onclick="selectenable()">disable' +
 	'<br>&nbsp;&nbsp;*index:'+
-	'<input class="button value" type="text" name="index" id="index">'+
+	'<input class="button value" type="text" name="index" id="index"  onkeydown=octkeydown(this) onkeyup=octkeyup(this)>'+
 	'<br>&nbsp;&nbsp;&nbsp;cir:'+
 	'<input class="button value" type="text" name="cir" id="cir" onkeydown=octkeydown(this) onkeyup=octkeyup(this)>'+
 	'<br>&nbsp;&nbsp;&nbsp;cbs:'+
-- 
2.17.1

